README.md

```diff
@@ -34,19 +34,19 @@ import okhttp3.OkHttpClient
 You can create an AuthHelper and use it with the AWS Kotlin SDK:
 
 ```
-// Create an authentication helper instance using an Amazon Location API Key
-private fun exampleAPIKeyLogin() {
+// Create a credentail provider using Identity Pool Id with AuthHelper
+private fun exampleCognitoLogin() {
     var authHelper = AuthHelper(applicationContext)
-    var locationCredentialsProvider : LocationCredentialsProvider = authHelper.authenticateWithApiKey("My-Amazon-Location-API-Key")
+    var locationCredentialsProvider : LocationCredentialsProvider = authHelper.authenticateWithCognitoIdentityPool("My-Cognito-Identity-Pool-Id")
     var locationClient = locationCredentialsProvider?.getLocationClient()
 }
-```
 
-```
-// Create an authentication helper using credentials from Cognito
-private fun exampleCognitoLogin() {
+OR
+
+// Create a credentail provider using custom credential provider with AuthHelper
+private fun exampleCustomCredentialLogin() {
     var authHelper = AuthHelper(applicationContext)
-    var locationCredentialsProvider : LocationCredentialsProvider = authHelper.authenticateWithCognitoIdentityPool("My-Cognito-Identity-Pool-Id")
+    var locationCredentialsProvider : LocationCredentialsProvider = authHelper.authenticateWithCredentialsProvider("MY-AWS-REGION", "MY-CUSTOM-CREDENTIAL-PROVIDER")
```

Comment:
The second argument in the `authenticateWithCredentialsProvider` is not as string, it is a `CredentialsProvider`

Reply:
Resolved